foundation/foundation-mda/src/main/java/com/boozallen/aiops/mda/metamodel/element/java/JavaStepDataRecordType.java

```diff
@@ -41,7 +38,7 @@ public String getFullyQualifiedType() {
         String fullyQualifiedRecordType = null;
 
         String packageName = getPackage();
-        if (StringUtils.isNotBlank(packageName) && !packageName.equals(modelRepository.getBasePackage())) {
+        if (StringUtils.isNotBlank(packageName) ){
```

Comment:
I: Not sure this is intended but it was initially there as part of 0.20.0 milestone to make recordtype functionality.

Reply:
The original thought process was probably that we don't need an import if it's in the same package.  Which _should_ still be true.  If we remove this and that results in an import, then the import statement would be harmless but also show up as unnecessary in most IDEs.

Reply:
Interesting, thought it would alway match base model and pipeline package. Good to know, Thank you!